pygmt/helpers/utils.py

```diff
@@ -131,6 +131,11 @@ def build_arg_string(kwargs):
     same command line argument. For example, the kwargs entry ``'B': ['xa',
     'yaf']`` will be converted to ``-Bxa -Byaf`` in the argument string.
 
+    Note that spaces ` ` in arguments are converted to the equivalent octal
+    code `\040`, except in the case of -J (projection) arguments where PROJ4
```

Comment:
It's ReST, not markdown.
```suggestion
    Note that spaces `` `` in arguments are converted to the equivalent octal
    code ``\040``, except in the case of -J (projection) arguments where PROJ4
```